src/Symfony/Component/Validator/Constraints/Timezone.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+* This file is part of the Symfony package.
+*
+* (c) Fabien Potencier <fabien@symfony.com>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Mickaël Andrieu <mickael.andrieu@sensiolabs.com>
```

Comment:
was from sensiolabs when I did this pull request.
Extracted from a project with project manager agreement ;)